spire-flex/values.schema.json

```diff
@@ -158,80 +143,104 @@
                                 "1.8.0",
                                 "1.7.2"
                             ]
+
                         }
                     }
-                }
```

Comment:
I cut most of the "compound" examples where multiple items were shown together.  It was (due to its JSON roots) getting too long to easily edit.